### ISSUE TITLE
Hardware/reference/daisy_seed/ES_Daisy_Seed_Rev4.pdf: No STM32 part number

### ISSUE THREAD

**Issue by anthroid** [electro-smith/DaisyExamples#103]
Hardware/reference/daisy_seed/ES_Daisy_Seed_Rev4.pdf: No STM32 part number. This should ideally be printed on Sheet 2 of ES_Daisy_Seed_Rev4.pdf. Ideally also mentioned somewhere in the README.md or the wiki.

**Comment by MrBlueXav** [comment]
Yes, please write part numbers on schematics. We could also have a BOM ?
*** STM32H750IBK6 ***
